app/assets/javascripts/sample_move.js

```diff
@@ -1,37 +0,0 @@
-function createNameAsset()
```

Comment:
There's quite a bit of code deletion going on here, as I ended up hitting large chunks of the codebase when searching for roles. Stripped out unused stuff, and tried to cascade it out. Hence why I'm removing some javascript.